src/SlackConnector/Connections/Sockets/Messages/Inbound/MessageInterpreter.cs

```diff
@@ -1,28 +1,33 @@
 ﻿using System;
 using System.Net;
 using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
 
 namespace SlackConnector.Connections.Sockets.Messages.Inbound
 {
     internal class MessageInterpreter : IMessageInterpreter
     {
         public InboundMessage InterpretMessage(string json)
         {
-            InboundMessage message = null;
-
             try
             {
-                message = JsonConvert.DeserializeObject<InboundMessage>(json);
+                var message = JsonConvert.DeserializeObject<InboundMessage>(json);
```

Comment:
I haven't tested it extensively, but it seemed more efficient, as I only deserialize a part of the message, compared to the JObject (which will represent the whole json)

http://www.newtonsoft.com/json/help/html/SerializingJSONFragments.htm